baton/handler.go

```diff
@@ -27,5 +30,7 @@ func serveWs(w http.ResponseWriter, r *http.Request) {
 	c := conn{send: make(chan []byte, 256), ws: ws}
 	h.register <- c
 	go c.writePump()
+	c.send <- []byte(strconv.Itoa(i))
```

Comment:
Might it make sense to send this in an initialization JSON object? Something like:

```js
{
  "type": "initialization",
  "data": {
    "id": [id]
  }
}
```

Not saying that should be the format of the object, but just wondering if there should be an object sent at all instead of a raw number.